index.html

```diff
@@ -48,186 +57,220 @@ <h3><a href="#">LET'S CONNECT</a></h3>
-            <h2>Languages <span class="arrow-down"><i class="fa-solid fa-caret-down"></i></span></h2>
-            <ul>
-                <li><a href="#"><i class="fa-brands fa-square-js"></i> Javascript</a></li>
-                <li><a href="#"><i class="fa-brands fa-html5"></i> HTML</a></li>
-                <li><a href="#"><i class="fa-brands fa-css3-alt"></i> CSS</a></li>
-            </ul>
-        </div>
-        <div class="about-frame">
-            <ul>
-                <li>Frameworks <span class="arrow"><i class="fa-solid fa-caret-right"></i></span></i></li><hr>
-                <li>Skills <span class="arrow1"><i class="fa-solid fa-caret-right"></i></span></li><hr>
-            </ul>
         </div>
 
     </section>
-    <section class="form">
-        <div class="form-heading">
-            <h1>Contact me</h1>
-        </div>
-        <div class="form-description">
-            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-        </div>
-        <div class="form-work">
-            <form action="https://formspree.io/f/xeqwgzag" method="POST">
-                <ul>
-                    <li>
-                        <input type="text" name="Username" id="name" minlength="6" maxlength="30" required placeholder="Full Name">
-                    </li>
-                    <li>
-                        <input type="email" name="Email" id="mail" required placeholder="Email" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$]">
-                    </li>
-                    <li>
-                        <textarea name="Message" id="msg" cols="30" rows="10" required placeholder="Write Your message here." maxlength="500"></textarea>
-                    </li>
-                    <li>
-                        <input type="submit" name="submit" id="send" value="Get in touch">
-                    </li>
-                </ul>
-            </form>
+    <section class="form" id="contact">
+        <div class="form-container">
+            <div class="form-heading">
+                <h1>Contact me</h1>
+            </div>
+            <div class="form-description">
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            </div>
+            <div class="form-work">
+                <form action="https://formspree.io/f/xeqwgzag" method="POST">
+                    <ul>
+                        <li>
+                            <input type="text" name="Username" id="name" minlength="6" maxlength="30" required placeholder="Full Name">
+                        </li>
+                        <li>
+                            <input type="email" name="Email" id="mail" required placeholder="Email" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$]">
+                        </li>
+                        <li>
```

Comment:
- [ ] I think it would be better also to add the correct background image as given by the Figma design ✨

![image](https://user-images.githubusercontent.com/102757126/220879800-2dc5fe10-f7b9-4263-ba1f-cc7f15ab3d33.png)